.github/jobs/build_docker_image.sh

```diff
@@ -2,7 +2,7 @@
 
 source ${GITHUB_WORKSPACE}/.github/jobs/bash_functions.sh
 
-DOCKERHUB_TAG=${DOCKERHUB_REPO}:${SOURCE_BRANCH}
+DOCKERHUB_TAG=get_dockerhub_tag
```

Comment:
```suggestion
DOCKERHUB_TAG=`get_dockerhub_tag`
```
With the above changes, the function will echo the value and this change will set the echo'd value to the DOCKERHUB_TAG variable.